### PR TITLE
Installation: Doc and license-less license clarifications

### PR DESCRIPTION
**Pull request by MaelRL**
Add a few lines about the license of the documentation and of files without copyright or license.

A few related improvements on the way.

## Release Management

* Affected package(s): `Installation`
* License and copyright ownership: no changes

### CODE REVIEW

**Review by sloriot — APPROVED** [review]

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.2-Ic-42.shtml

**Comment by lrineau** [comment]
There is an ongoing (slow) discussion in https://github.com/CGAL/cgal/issues/5140. I will not merge this PR before it is over.

File: Documentation/doc/Documentation/License.txt [hunk 1]
```diff
@@ -69,36 +67,16 @@ To enable checking, users have to define one of the following macros:
...
-`CGAL_VERSION_STR`, but not as a string literal.
+\section licensesWhere Determining the license of a CGAL file
+
+It is specified in each file of the \cgal library which license applies to it.
+
+The \cgal documentation is distributed under the Creative Commons
+<a href="https://creativecommons.org/licenses/by-nc-nd/2.0/">BY-NC-ND</a> License.
+
+All files that do not have an explicit copyright or license notice (e.g., all
+examples, some demos, data files without a license, ...) are licensed under a very permissive license.
```
Comment (afabri) [inline Documentation/doc/Documentation/License.txt:78]: Here should refer to the [CC0 ](https://creativecommons.org/publicdomain/zero/1.0/legalcode)and remove the `LICENSE.FREE_USE`

Comment (afabri) [reply Documentation/doc/Documentation/License.txt:78]: @MaelRL this closes the discussion.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-3.shtml

**Comment by afabri** [comment]
> 
> 
> Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-3.shtml

The decided change has not been done yet.

**Comment by maxGimeno** [comment]
ping @MaelRL

**Comment by afabri** [comment]
@maxGimeno  we only have to look if the link to  the CC0 is correct in the manual.

**Comment by sloriot** [comment]
Doc build failed

**Comment by maxGimeno** [comment]
I think it is because it doesn't touch any package, so it was discarded...

**Comment by maxGimeno** [comment]
More precisely, it is because DOcumentation doesn't have a /package_info, which is a condition to be in the list of "packages" tested by the script. If this list is empty, then it just stops.

**Comment by lrineau** [comment]
The content of two files is currently wrong:
  - `/LICENSE.md` (says all files without license are GPLv3+), and
  - `/Installation/LICENSE` (mentions `LICENSE.FREE_USE`).

**Comment by lrineau** [comment]
Besides, there is now a merge conflict in `Installation/CMakeLists.txt`, @maxGimeno.

**Comment by afabri** [comment]
I think you should write `CC0 1.0`   as I did in my commit (in doc, that is I missed to change the top level files)

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-11.shtml

**Comment by lrineau** [comment]
Oops! This PR brokes the generation of CGAL Windows installer:
```
File: "INSTALL.md" [compress] 317/462 bytes
File: "LICENSE" [compress] 929/2175 bytes
File: "CGAL-5.2\LICENSE.FREE_USE" -> no files found.
Usage: File [/nonfatal] [/a] ([/r] [/x filespec [...]] filespec [...] |
   /oname=outfile one_file_only)
Error in script "script_cgal.nsi" on line 174 -- aborting creation process
```

Now that so many things are automatized, I had not check that correctly. So, CGAL-5.2 has been published without the Windows installer.

**Pull request merged**